rio/srcrobot/constants.py

```diff
@@ -202,3 +202,13 @@ def __init__(self, value, blueSideBotHeading, redSideBotHeading, armAngle, shoot
         self.m_redSideBotHeading = redSideBotHeading
         self.m_armAngle = armAngle
         self.m_shooterVelocity = shooterVelocity
+
+      def calculate(self, distance):
```

Comment:
BTW, when we are ready to try this, we can add a shot to the enum to represent this and augment the execute trigger to run a different smart implementation when this is the selected shot.